src/Runner.Listener/Runner.cs

```diff
@@ -510,14 +511,17 @@ private void PrintUsage(CommandSettings command)
  --runnergroup string   Name of the runner group to add this runner to (defaults to the default runner group)
  --labels string        Extra labels in addition to the default: 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'
  --work string          Relative runner work directory (default {Constants.Path.WorkDirectory})
- --replace              Replace any existing runner with the same name (default false)");
+ --replace              Replace any existing runner with the same name (default false)
+ --pat                  GitHub personal access token used for checking network connectivity when executing `.{separator}run.{ext} --check`");
```

Comment:
Should this should the `{separator}run{ext}` - i.e. remove the `.`?

Reply:
same question further below on line 523

Reply:
we have the `.` in everywhere else. 😄